sp800-63d/vot_mapping.md

```diff
@@ -118,25 +118,87 @@ FAL maps into the VoT component A using a numeric identifier. Only one numeric i
+* Single-Factor Cryptographic Device used in conjunction with Memorized Secret
+
+If the user authenticates using a restricted authenticator, Cr and the value representing the authenticator used SHALL be indicated in the vector.
+
+**Example:** Cr.Co would be asserted if the user authenticates using a secret sent via SMS to an out-of-band device.
+
+### Authentication Characteristics
+
+If implementing AAL1 or AAL2, and:
+
+* the user authenticates with an authenticator that provides verifier impersonation resistance, Cv SHOULD be indicated in the vector.
+* the user authenticates with an authenticator that provides verifier compromise resistance, Cs SHOULD be indicated in the vector.
+* authentication intent from at least one authenticator is demonstrated, Cn SHOULD be indicated in the vector.
+* for non-government agency authenticators and verifiers, authenticator(s) or verifier(s) are validated to meet the requirements of FIPS 140 Level 1, Ci SHOULD be indicated in the vector.
+
+If using a biometric for authentication, and:
+
+* biometric comparison is performed centrally (not locally on the claimant's device), Ck SHALL be indicated in the vector.
+* the biometric system implements PAD, Ck SHOULD be indicated in the vector.
+
+**NOTE: Should we add a vector value for FMR?**
+
+MitM attack resistance
+
+**NOTE: I'm struggling to find the case where a MitM resistance value is interesting - it seems to be required most of the time and would be redundant with the xAL indicated in the vector.**
+
+### Authenticator Lifecycle Management
+
+If the authenticator(s) is/are bound/issued:
+
+* during a single proofing session, Mp SHALL be indicated in the vector.
+* remotely after the proofing session, Mr SHALL be indicated in the vector.
+* in-person after the proofing session, Mi SHALL be indicated in the vector.
+
+If the subscriber's account had only one authentication factor bound to it at account creation, and an additional authenticator of a different authentication factor is added after account creation, Ms SHALL be indicated in the vector.
+
+**NOTE: I don't think the phrasing is quite right on adding a second factor yet.**
+
+If a subscriber loses all authentictors of a factor necessary to complete multi-factor authentication, has been identity proofed at IAL2 or IAL3, and consequently undergoes an abbreviated identity proofing process to reestablish authentication factors, Ma SHALL be indicated in the vector.
+
+### Assertion Presentation
+
+If a front-channel presentation mechanism is used, Af SHOULD be indicated in the vector.
+
+If a back-channel presentation mechanism is used, Ab SHOULD be indicated in the vector.
+
+**NOTE: We could build a table to help readers distinguish between values that are required in the vector (SHALLs/depending on implementation) vs. encouraged (SHOULDs/depending on implementation). I can also add in references to A-C as relevant/appropriate so readers can easily find related guidance.**
```

Comment:
References are definitely good and a table might help wade through the sea of requirements. In other spaces I've seen formatted lists instead, but that's also using the IETF normative syntax which gives use "REQUIRED" and "OPTIONAL" in addition to "SHALL" and "SHOULD"